crates/graph/src/tests/mod.rs

```diff
@@ -188,14 +187,6 @@ macro_rules! binary_test {
     };
 }
 
-#[allow(dead_code)]
-pub fn assert_op_in_graph<T: Operator + 'static>(graph: &Graph) {
-    assert!(
-        graph.node_indices().any(|i| graph.check_node_type::<T>(i)),
-        "Node not found in the graph!"
-    );
-}
-
```

Comment:
Not used anywhere. Shouldn't be too difficult to retrieve from history in the future if needed.